src/components/flatList.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ListRenderItem } from 'react-native';
+import { SafeAreaView } from 'react-native';
+import { StyleSheet, View, FlatList, Text, StatusBar } from 'react-native';
+import { Journey } from '../model/fareResponseClass';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    marginTop: StatusBar.currentHeight || 0,
+  },
+  item: {
+    backgroundColor: '#f9c2ff',
+    padding: 10,
+    marginVertical: 2,
+    marginHorizontal: 4,
+  },
+  title: {
+    fontSize: 14,
+  },
+});
+
+type FlatListProps = {
+  items: Journey[] | null;
```

Comment:
To me, it doesn't really feel like it makes sense to be able to pass null items to this component. Its whole purpose is to display a list of items, so it doesn't really make sense to try to create it if we know we don't have any items.

It would be better to handle the nullability outside the component in the parent and not create this component if items is null